themes/default/content/case-studies/lykke.md

```diff
@@ -0,0 +1,67 @@
+
+customer_name: Lykke
+customer_logo: /logos/customers/lykke_logo.svg
+customer_url: https://lykke.com/
+
+exec_summary: |
+    [Lykke](https://lykke.com/) is a Swiss fintech company bridging the gap between traditional finance and Blockchain. Lykke operates a commission-free exchange for digital assets and cryptocurrencies. Lykke developers rely on scalable blockchain infrastructure and leverage Pulumi to define reusable components and services and create abstractions needed to support its ecosystem of products. Using Pulumi, Lykke has been able to rapidly lower the time-to-market for its services and deliver innovative new capabilities for the fast-growing cryptocurrency market.
+
+sections:
+    - label: Exec Summary
+      anchor: executive-summary
+    - label: About Lykke
+      anchor: about-lykke
+    - label: Challenges Faced
+      anchor: challenges-faced
+    - label: Results
+      anchor: results
+    - label: Next Steps
+      anchor: next-steps
+---
+
+## About Lykke
+
+Founded in 2013, Lykke received its initial funding in 2015 to bridge the gap between traditional finance and blockchain. The company builds on decades of research by its founder Richard Olsen, a pioneer in the field of high-frequency finance. Its vision is to provide a global marketplace where financial instruments can be traded and settled peer-to-peer with second-by-second interest payments. This open-source marketplace utilizes blockchain technology pioneered by Bitcoin to offer immediate settlement and direct ownership without transaction fees.
+
+## Challenges Faced
+
+Lykke originally built a monolithic stack with a wide variety of components. The engineering team wanted to adopt a serverless architecture and in April 2019, they decided to build a completely new infrastructure to support new applications - all without a dedicated ops team. To add to the complexity, the group needed to accurately model key parts of their infrastructure and create abstractions that would simplify scaling and make it accessible to both-back end and front-end engineers. This greenfield project was an exciting opportunity to select best-in-class solutions that worked well with their chosen cloud provider, AWS, and were compatible with their preferred engineering tools and frameworks -- React Native, .NET Core, and serverless architecture.
+
+## Solution: Pulumi Crosswalk and Infrastructure-as-Code
+
+Lykke engineering head Damian Hickey had been aware of Pulumi since the company launched in 2017 and decided to deploy the new infrastructure using [Pulumi Crosswalk for AWS]({{< relref "/crosswalk/aws" >}}). This allowed him to design and manage infrastructure within his development team, rather than building a dedicated ops team. Crosswalk also provided Lykke with built-in Well Architected patterns and best practices - helping the team move new applications into production faster.
```

Comment:
```suggestion
Lykke engineering head Damian Hickey had been aware of Pulumi since the company launched in 2017 and decided to deploy the new infrastructure using [Pulumi's collection of libraries]({{< relref "/crosswalk/aws" >}}) that model AWS infrastructure patterns using well-architected best practices. This allowed him to design and manage infrastructure within his development team, rather than building a dedicated ops team. Pulumi's libraries also provided Lykke with built-in Well Architected patterns and best practices - helping the team move new applications into production faster.
```